src/api/spec/presenters/comments_controller_presenters/comment_presenter_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe ::CommentsControllerPresenters::CommentPresenter do
+  let(:user) { create(:confirmed_user, login: 'tom') }
+  let(:comment_project) { create(:comment_project, user: user) }
+  let(:comment_package) { create(:comment_package, user: user) }
+
+  describe 'obj is an user' do
```

Comment:
I would use a more descriptive term here, like if `obj` is the comenter, I would write `Commenter is a ...`

Reply:
thanks @bgeuken, done!